package.json

```diff
@@ -139,7 +139,10 @@
             "platform": "ios",
             "appRoot": "${workspaceRoot}",
             "sourceMaps": true,
-            "watch": true
+            "watch": true,
+            "sourceMapPathOverrides": {
+               "webpack:///*": "${workspaceRoot}/*"
```

Comment:
If we'll stick with `inline-source-map` option, should be `"webpack:///*": "${workspaceRoot}/app/*"`